yarn.lock

```diff
@@ -131,25 +131,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@auth/core@npm:^0.1.4":
```

Comment:
Not sure if we want to let update the entire `yarn.lock` when adding a new dependency, maybe just pick up the changes to introduce the dependency and leave the rest out for its own updating process?